lib/public/app.css

```diff
@@ -665,3 +665,13 @@ label {
         grid-template-columns: repeat(3, 1fr);
     }
 }
+
+.run-quality.good {
+    background-color: var(--color-green) !important;
+    color: var(--white);
+}
+
+.run-quality.bad {
+    background-color: var(--color-soft-red) !important;
+    color: var(--white);
+}
```

Comment:
I think this is too specific. You are mixing business stuff and style sheet, why not using `.white.bg-green` for example?

Reply:
refactored